src/components/List/List.jsx

```diff
@@ -102,12 +107,15 @@ const defaultProps = {
   pagination: null,
   selectedIds: [],
   expandedIds: [],
+  items: [],
   handleSelect: () => {},
   toggleExpansion: () => {},
   onItemMoved: () => {},
   itemWillMove: () => {
     return true;
   },
+  hasEmptyState: false,
+  emptyState: 'Empty list',
```

Comment:
```suggestion
  emptyState: 'No list items to show',
```

Loosely based on [the guidance from the site](https://pages.github.ibm.com/ai-applications/design/components/empty-states/usage#data-missing)